TACC/Core-Styles
# fix: admonition type "tip" has no color

## Overview

Make sure `.admonition.tip` has color via `message-extra`.

## Related

- might have already been fixed by #373
    <sup>testing was confusing; I am tired</sup>

## Changes

- **renamed** `message-tip` to `message-extra`
- **added** missing custom selector for `…extra` f.k.a `…tip`

## Testing

1. Open http://localhost:3000/components/preview/admonition.
2. Verify "Tip" admonition has blue background.

## UI

| before | after |
| - | - |
| <img width="1024" alt="before" src="https://github.com/user-attachments/assets/ed9a2923-98b8-43bc-93fb-fa581edebc20"> | <img width="1024" alt="after" src="https://github.com/user-attachments/assets/76f3c501-bf65-481b-8495-719f4353eca1"> |
